File: examples/javascript/handler.js
// JavaScript example function for GraalFaaS
// Contract: export a global function named `handler(event)` that returns a JSON-serializable value.
function handler(event) {
  const name = event && event.name != null ? event.name : "World";
  return { message: `Hello, ${name}!` };
}

File: examples/python/handler.py
# Python example function for GraalFaaS
# Contract: define a function `handler(event)` that returns a JSON-serializable value.

def handler(event):
    try:
        name = event.get("name", "World") if isinstance(event, dict) else "World"
    except Exception:
        name = "World"
    return {"message": f"Hello, {name}!"}
